Example/Emission/AppDelegate.m

```diff
@@ -191,7 +191,7 @@ - (UIViewController *)viewControllerForRoute:(NSString *)route;
 {
   UIViewController *viewController = nil;
 
-  if ([route hasPrefix:@"/artist/"] && [route componentsSeparatedByString:@"/"].count == 3) {
+  if ([route hasPrefix:@"/artist/"] && [route componentsSeparatedByString:@"/"].count == 1) {
```

Comment:
Woops - I fixed this on the previous works-for-you branch but forgot about it here.